#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
uint8 left_front
uint8 left_rear
uint8 right_front
uint8 right_rear
uint8 pod_speed
